src/oge/create_nox_so2_factors.py

```diff
@@ -0,0 +1,574 @@
+import pandas as pd
+
+from oge.download_data import download_helper
+from oge.filepaths import downloads_folder
+from oge.logging_util import get_logger
+from oge.constants import ConversionFactors
+
+logger = get_logger(__name__)
+
+
+def generate_nox_emission_factor_reference_table() -> pd.DataFrame:
+    """Coordinating function for creating the reference table for NOx uncontrolled
+    emission factors. This is run in notebooks/manual_data/
+    create_nox_so2_factor_tables.ipynb.
+
+    We download "Table A.2. Nitrogen Oxides Uncontrolled Emission Factors" from the EIA,
+    standardize units, and fill in specific boiler configurations that are not
+    explicitly enumerated in the EIA table.
+
+    This table from the EIA has a row for earch energy_source_code, and the columns
+    represent factors specific to a certain boiler firing type (BFT) or prime-mover (PM)
+    PM-specific factors are provided only for combustion turbine and internal combustion
+    engines. For certain BFTs (tangential boilers and all other boilers), this table
+    additionally differentiates between dry-bottom boilers and wet-bottom boilers.
+
+    Returns:
+        pd.DataFrame: Uncontrolled NOx emission factors for all combinations of
+            energy_source_code, prime_mover, boiler_firing_type, and wet_dry_bottom.
+    """
+    # download the nox data from EIA
+    download_helper(
+        "https://www.eia.gov/electricity/annual/xls/epa_a_02.xlsx",
+        downloads_folder("eia/table_a2_nox_uncontrolled_emission_factors.xlsx"),
+    )
+
+    nox_factors_eia = pd.read_excel(
+        downloads_folder("eia/table_a2_nox_uncontrolled_emission_factors.xlsx"),
+        header=3,
+        names=[
+            "fuel",
+            "energy_source_code",
+            "data_source",
+            "unit",
+            "cyclone_firing",
+            "fluidized_bed_firing",
+            "stoker",
+            "tangential_firing_dry",
+            "tangential_firing_wet",
+            "other_dry",
+            "other_wet",
+            "GT",
```

Comment:
Should it be `CT` for combustion turbine? Just want to make sure it is not a typo

Reply:
If you look at the prime mover metadata (https://catalystcoop-pudl.readthedocs.io/en/latest/data_dictionaries/codes_and_labels.html#core-eia-codes-prime-movers), GT is the code for a general combustion turbine, while CT is specifically a combustion turbine that is part of a combined cycle unit. 
We end up applying this EF to the CT prime mover later in the pipeline.